docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3.8'
+
+services:
+  be-app:
+    image: demo
+    build: .
+    ports:
+      - "80:8080"
+    restart: always
+    environment:
+      DATASOURCE_URL: jdbc:postgresql://localhost:5432/foodbe-db
+      ENABLE_LOG: true
```

Comment:
- Lack of DB_USERNAME and DB_PASSWORD, if you don't set the ENV, there may be some errors when running the docker-compose
- This docker-compose.yml should be full of env, or something needed for building the image because I will not see all of the env